### ISSUE TITLE
Última versão do Software do Cartão de Cidadão para Ubuntu 16.04

### ISSUE THREAD

**Issue by TiagoMauricio** [amagovpt/autenticacao.gov#7]
Bom dia,

Pretendo instalar o software do cartão de cidadão numa máquina com Ubuntu 16.04.
Na lista oficial sistemas operativos suportados ( https://www.autenticacao.gov.pt/cc-software ), apenas está listado o software para Ubuntu 18.04 ou superior.

Poderiam indicar-me onde posso encontrar o software para Ubuntu 16.04?

Muito obrigado.

**Comment by agrr** [comment]
A última versão do software que foi lançada para Ubuntu 16.04 (2.4.6) ainda está disponível no antigo repositório do projeto:
https://svn.gov.pt/projects/ccidadao/repository/middleware-offline/tags/builds/lastversion/
Poderá servir para algum propósito, mas atenção que não irá ler cartões recentes nem deverá funcionar a confirmação de morada.

Esse S.O. não tem a versão de QT mínima que a aplicação requer neste momento (5.9) daí não ter suporte desde a versão 3.

**Comment by TiagoMauricio** [comment]
Muito obrigado pela informação.

Se instalar a versão do QT necessária manualmente é possível fazer build do source code?

Obrigado

**Comment by agrr** [comment]
Sim, deverá ser possível usando os comandos indicados no README.md do projeto, ignorando a instalação de pacotes do QT5 a partir dos repos do Ubuntu.

**Comment by TiagoMauricio** [comment]
Bom dia, no seguimento da minha questão considerei que a hipótese de instalar QT 5.9 em 16.04 é complicado. Estou neste momento a fazer testes com uma versão de 20.04 e estou a ter problemas de dependências. Tem alguma sugestão sobre como solucionar o problema das dependências?

` Unpacking pteid-mw (3.0.20+git20190930-e547bd9d-1) ...
dpkg: dependency problems prevent configuration of pteid-mw:
 pteid-mw depends on libssl1.0.0 (>= 1.0.2~beta3); however:
  Package libssl1.0.0 is not installed.
 pteid-mw depends on libxml-security-c17v5; however:
  Package libxml-security-c17v5 is not installed.
 pteid-mw depends on libzip4 (>= 1.0); however:
  Package libzip4 is not installed.
 pteid-mw depends on qtbase-abi-5-9-5; however:
  Package qtbase-abi-5-9-5 is not installed. `
